src/scripts/glue_jobs/glue_job.py

```diff
@@ -54,16 +56,15 @@ def create_input_frame_from_s3(self):
             # Note: even though this is optional, job bookmark does not work without it
             transformation_ctx="input_frame"
         )
-        self.logger.info("Total input records read from s3 for job {} is {}".format(self.args["JOB_NAME"],
-                                                                                    str(input_frame.count())))
+        self.check_and_log_errors(input_frame)
+        self.logger.info("The total number of records retrieved from s3 is {}".format(input_frame.count()))
```

Comment:
I would leave this out since this slows down each job and does not provide any actionable information